package.json

```diff
@@ -29,44 +29,41 @@
     "react-dom": "^15.0.0-0 || ^16.0.0-0"
   },
   "devDependencies": {
-    "@storybook/react": "^3.1.7",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
+    "@storybook/addon-actions": "^4.1.18",
+    "@storybook/addon-links": "^6.0.26",
+    "@storybook/react": "^4.1.18",
     "chai": "^3.5.0",
     "copy-to-clipboard": "^3.0.8",
```

Comment:
Are you sure this is needed?

Reply:
It failed without it

Reply:
I remove addon-links

Reply:
really strange. Where is the error coming from?